robottelo/ui/locators/base.py

```diff
@@ -1630,7 +1630,7 @@
         "//u[contains(.,'%s')]/../../div/form/div/select"),
     "ak.subscriptions.search": (
         By.XPATH,
-        "//input[@ng-model='subscriptionSearch']"),
+        "//input[@ng-model='table.searchTerm']"),
```

Comment:
Maybe it would be better to replace references to this locator with `common_locators["kt_search"]`?